### ISSUE TITLE
Two DLLs trying to be copied with the same name

### ISSUE THREAD

**Issue by bryaan** [microsoft/CsWinRT#1089]
I am getting the following build error:

```
 Microsoft.Build.Msix.Packaging.targets(1370, 5): [APPX1101] Payload contains two or more files with the same destination path 'Microsoft.Web.WebView2.Core.dll'. Source files: 
C:\Users\Bryan\.nuget\packages\microsoft.web.webview2\1.0.1018-prerelease\lib\netcoreapp3.0\Microsoft.Web.WebView2.Core.dll
C:\Users\Bryan\.nuget\packages\microsoft.web.webview2\1.0.1018-prerelease\runtimes\win-x64\native_uap\Microsoft.Web.WebView2.Core.dll

```
This happened after installing the Microsoft.Windows.CsWinRT v1.4.1 package in my winui3 app and a winui3 class library.

**Comment by manodasanW** [comment]
@bryaan I haven't been able to repro this.  Can you provide a repro that I can try out?

**Comment by manodasanW** [comment]
Closing out old issues with no recent feedback.

**Comment by ChrisGuzak** [comment]
I can reproduce this and I'm sure everyone else can too. Create a WinAppSDK using the VS project template. I've tried the full App as well as the unit test project types. Both have the same issue. I tried updating all of the packages including using preview SDK tools and get the same result

I also found a work around, disable cswinrt integration using `<WebView2EnableCsWinRTProjection>false</WebView2EnableCsWinRTProjection>`. Adding that to `<PropertyGroup Label="Globals">`
